examples/client/lwm2mclient.c

```diff
@@ -665,6 +763,13 @@ static void update_battery_level(lwm2m_context_t * context)
     }
 }
 
+/**
+ * @brief Adds a test object to the LWM2M server.
```

Comment:
A small .md with this PR as a supplementary doc would be helpful. It would contain details explaining e.g. what is a object, what is a client what is a server in COAP definitions and how does this connect with current wakaama examples. Hence, this operation of adding a test object to the LWM2M server would be more clear for the user

Reply:
I agree, but I would say this goes beyond current task and its description. 
Generally speaking, we do not have comprehensive documentation and it could be different ticket to create that documentation with more context, beyond these inline comments. We can further discuss it

Reply:
Ok.